scripts/apps/authoring/authoring/controllers/ChangeImageController.ts

```diff
@@ -101,8 +101,16 @@ export function ChangeImageController($scope, gettext, notify, modal, _, api, $r
         }
     };
 
+    $scope.handleNoImageCrops = function(_origCropsData) {
```

Comment:
There is not much use of having the function if you are only using it once and it depends on the closure values. Can you inline it?

Reply:
You mean inline it, somewhere in the template?

Reply:
By inline, I mean to replace the function call with the contents of the function and remove the function then.

Reply:
fixed that.